microsoft-edge/webview2/release-notes.md

```diff
@@ -31,6 +31,159 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
+    get_ShouldPrintBackgrounds
+    put_ShouldPrintBackgrounds
+    get_ShouldPrintSelectionOnly
+    put_ShouldPrintSelectionOnly
+    get_ShouldPrintHeaderAndFooter
+    put_ShouldPrintHeaderAndFooter
+    get_HeaderTitle
+    put_HeaderTitle
+    get_FooterUri
+    put_FooterUri
+    get_UserDataFolder
+
+<!-- ====================================================================== -->
+## 1.0.1083-prerelease
+  
+Release Date: November 29, 2021  
+  
+[NuGet package for WebView2 SDK 1.0.1083](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1083-prerelease)  
+  
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 97.0.1083.0 or higher.  
+
+### General
+
+#### Experimental Features
+  
+ICoreWebView2ExperimentalFrame2
+(ADDED)
+    PostWebMessageAsJson
+    PostWebMessageAsString
+    add_WebMessageReceived
+    remove_WebMessageReceived
+ICoreWebView2ExperimentalFrameWebMessageReceivedEventHandler
+ (ADDED)
+ICoreWebView2ExperimentalProcessInfo 
+(ADDED)
+    get_ProcessId
+    get_Kind
+ICoreWebView2ExperimentalProcessInfoCollection 
+(ADDED)
+    get_Count
+    GetValueAtIndex
+ICoreWebView2ExperimentalProcessInfosChangedEventHandler 
+(ADDED)
+ICoreWebView2ExperimentalNavigationStartingEventArgs (ADDED)
+ICoreWebView2Experimental9 
+(REMOVED)
+    add_IsMutedChanged
+    remove_IsMutedChanged
+    get_IsMuted
+    put_IsMuted
+    add_IsDocumentPlayingAudioChanged
+    remove_IsDocumentPlayingAudioChanged
+    get_IsDocumentPlayingAudio
+    add_IsDefaultDownloadDialogOpenChanged
+    remove_IsDefaultDownloadDialogOpenChanged
+    get_IsDefaultDownloadDialogOpen
+    OpenDefaultDownloadDialog
+    CloseDefaultDownloadDialog
+    get_DefaultDownloadDialogCornerAlignment
+    put_DefaultDownloadDialogCornerAlignment
+    get_DefaultDownloadDialogMargin
+    put_DefaultDownloadDialogMargin
+ICoreWebView2ExperimentalIsDefaultDownloadDialogOpenChangedEventHandler
+ (REMOVED)
+ICoreWebView2ExperimentalIsDocumentPlayingAudioChangedEventHandler 
+(REMOVED)
+ICoreWebView2ExperimentalIsMutedChangedEventHandler (REMOVED)
+
+#### Promotions
+
+The following APIs are promoted to stable in this prerelease SDK:  
+
+ICoreWebView2_8 
+(ADDED)
+    add_IsMutedChanged
+    remove_IsMutedChanged
+    get_IsMuted
+    put_IsMuted
+    add_IsDocumentPlayingAudioChanged
+    remove_IsDocumentPlayingAudioChanged
+    get_IsDocumentPlayingAudio
+ICoreWebView2_9
+ 
+(ADDED)
+    add_IsDefaultDownloadDialogOpenChanged
+    remove_IsDefaultDownloadDialogOpenChanged
+    get_IsDefaultDownloadDialogOpen
+    OpenDefaultDownloadDialog
+    CloseDefaultDownloadDialog
+    get_DefaultDownloadDialogCornerAlignment
+    put_DefaultDownloadDialogCornerAlignment
+    get_DefaultDownloadDialogMargin
+    put_DefaultDownloadDialogMargin
+ICoreWebView2IsDefaultDownloadDialogOpenChangedEventHandler
+ (ADDED)
+ICoreWebView2IsDocumentPlayingAudioChangedEventHandler (ADDED)
+ICoreWebView2IsMutedChangedEventHandler 
+(ADDED)
+
```

Comment:
From lines 152 to 177 there are 5 mentions of (ADDED) and they are all in different places. Need to be used consistently and clear what was added.

Reply:
@jasonstephen15 - Does this notation mean that ICoreWebView2ExperimentalFrameWebMessageReceivedEventHandler is entirely added?  No methods are listed for that interface.

```
ICoreWebView2ExperimentalFrame2
(ADDED)
    PostWebMessageAsJson
    PostWebMessageAsString
    add_WebMessageReceived
    remove_WebMessageReceived
ICoreWebView2ExperimentalFrameWebMessageReceivedEventHandler
 (ADDED)
ICoreWebView2ExperimentalProcessInfo 
(ADDED)
    get_ProcessId
    get_Kind
```

Reply:
reformatted these